### PR TITLE
Fixing ssl required for organization settings and index

### LINKED ISSUE #3115: Organization index and settings are not ssl required
cc @Kartones

### PR DESCRIPTION
**Pull request by xavijam**
REVIEWER: @Kartones 

Fixes #3115.

### CODE REVIEW

File: app/controllers/superadmin/organizations_controller.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Superadmin::OrganizationsController < Superadmin::SuperadminController
   respond_to :json
 
-  ssl_required :show, :create, :update, :destroy, :index if Rails.env.production? || Rails.env.staging?
```
Comment (xavijam) [inline app/controllers/superadmin/organizations_controller.rb]: i don't understand this thing, why should that conditional be needed?

File: app/controllers/admin/organizations_controller.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # coding: utf-8
 class Admin::OrganizationsController < ApplicationController
-  ssl_required :oauth, :api_key, :regenerate_api_key
```
Comment (xavijam) [inline app/controllers/admin/organizations_controller.rb]: We are genius :DDDD

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2919/))

File: app/controllers/superadmin/organizations_controller.rb (hunk 1 above)
Comment (Kartones) [inline app/controllers/superadmin/organizations_controller.rb]: Shouldn't in fact, there's an override for ssl_required that in "dev mode" sets it to false . This probably is old code

**Comment by Kartones** [comment]
:+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2922/))

**Pull request merged**